Cargo.toml

```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/PistonDevelopers/gfx_text.git"
 documentation = "http://docs.piston.rs/gfx_text/gfx_text/"
 
 [dependencies]
-gfx = "^0.5.2"
+gfx = "^0.6.0"
```

Comment:
`^0.6.0` is the same as `0.6.*` or even `0.6` but caret and tilda notations are used when patch number is not zero. Also, caret notation is implied by default if version specifier is omitted (`^0.6.0` = `0.6.0`) but I prefer to make it explicit because npm's (where the caret notation is from) `0.6.0` = `=0.6.0` in cargo and that may cause confusion.